frontend/src/Decorators.tsx

```diff
@@ -1,8 +1,12 @@
+/* eslint-disable react/no-multi-comp */
```

Comment:
Should add a comment why this is needed.

Reply:
I just added a comment. After this PR, as agreed with gabriele, we will do a "refactoring" of the decorators by dividing them into several files. This file will be removed.